plugins/wacom-i2c/fu-plugin-wacom-i2c.c

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-plugin-vfuncs.h"
+#include "fu-wac-aes-device.h"
+#include "fu-wac-emr-device.h"
+
+struct FuPluginData {
+	gchar			*technology;
+};
+
+void
+fu_plugin_init (FuPlugin *plugin)
+{
+	fu_plugin_alloc_data (plugin, sizeof (FuPluginData));
+	fu_plugin_add_udev_subsystem (plugin, "i2c");
+}
+
+void
+fu_plugin_destroy (FuPlugin *plugin)
+{
+	FuPluginData *priv = fu_plugin_get_data (plugin);
+	g_free (priv->technology);
+}
+
+gboolean
+fu_plugin_startup (FuPlugin *plugin, GError **error)
+{
+	FuPluginData *priv = fu_plugin_get_data (plugin);
+	GPtrArray *hwids = fu_plugin_get_hwids (plugin);
```

Comment:
Ahh, I've been talking to Wacom, and they're using the low-level i2c interface to update more hardware than just the Canvas. In that case we're matching on the system, as these are tablets that are glued into place, rather than tablets like covered in "wacomhid" which are USB devices.

Reply:
Okay that makes more sense.  I agree there should be devices that don't have the controller hooked up via USB that work this way.

Reply:
All nuked in favour of hidraw now.